teapots/choreographer-30fps/src/main/cpp/CMakeLists.txt

```diff
@@ -17,36 +17,53 @@
 cmake_minimum_required(VERSION 3.4.1)
 
 # build native_app_glue as a static lib
-add_library(native_app_glue STATIC
-    ${ANDROID_NDK}/sources/android/native_app_glue/android_native_app_glue.c)
-
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11 -Wall -fno-exceptions -fno-rtti")
```

Comment:
If I might do an addendum here:
if you define the flags (-fno-rtti against a common dependency such as ndkHelperLib) then the parent receives it automatically.  Thus no need for defining it in more than one place.
+1 for removing no-rtti/no-exceptions in a different change.

Reply:
this is one is done, will be there in the next upload!

Reply:
> if you define the flags (-fno-rtti against a common dependency such as ndkHelperLib) then the parent receives it automatically.

That's an odd behavior imo. Just because a library you use doesn't use RTTI/exceptions doesn't mean that your library shouldn't.

Regardless, it's less intention revealing. Adding it to `CXX_FLAGS` very obviously says "I want these flags applied to every C++ file".